Recommendation-System/clean.py

```diff
@@ -0,0 +1,84 @@
+# from _typeshed import NoneType
+import csv
+# import pandas as pd
+from io import StringIO
+from html.parser import HTMLParser
+import json
+import string
+import re
+# from datetime import datetime
+
+# startTime = datetime.now()
+
+printable = set(string.printable)
+
+# Opening JSON file
+f = open(r'C:\Users\devan\OneDrive\Desktop\Results\dataset.json', encoding="utf8")

```

Comment:
I think the file path given here should not be in codebase because the file path is your local environment specific and the file will not be fetched in the deployment environment.

Reply:
Yes Yes! You are absolutely right. This PR however is a draft PR, not the final one yet. I just wanted to move these codes from my local machine and @Shivansh2407 had suggested me to make a draft PR. I will edit all these things later on once the model is ready.